app/src/main/java/com/example/nvdovin/weatherapp/ForecastActivity.java

```diff
@@ -44,23 +44,26 @@ public void displayData(List<City> data) {
             builder.append(c.getName());
             builder.append("\n");
             builder.append(c.getLat());
-            builder.append("\n");
-            builder.append(c.getLat());
             builder.append(" ");
             builder.append(c.getLon());
             builder.append("\n");
             List<WeatherData> weatherDatas = c.getWeatherDataList();
-            builder.append(weatherDatas.get(0).getHumidity());
-            builder.append("\n");
-            builder.append(weatherDatas.get(0).getPressure());
-            builder.append("\n");
-            builder.append(weatherDatas.get(0).getWeather());
+            for (WeatherData w : weatherDatas) {
+                builder.append(w.getHumidity());
+                builder.append("\n");
```

Comment:
no matters what you will implement in future, if you commit your changes to repo they should be clear  without any commented methods or pieces of code